### PR TITLE
add the titlePlural property

### PR DESCRIPTION
**Pull request by tahmeedshah**
DEVDESK-547
BCH-Plural form of list of records

### CODE REVIEW

File: app/app-data/abs/realm-configuration.json [hunk 1]
```diff
@@ -134,7 +134,7 @@
           },
           "shortCode": "NDB",
           "titlePlural": {
-            "en": "National Websites or Databases"
+            "en": "National Biosafety Websites or Databases"
```
Comment (blaisefonseca) [inline app/app-data/abs/realm-configuration.json:137]: this abs file

File: app/app-data/abs/realm-configuration.json [hunk 2]
```diff
@@ -205,7 +205,7 @@
           },
           "shortCode": "ORG",
           "titlePlural": {
-            "en": "Organizations"
+            "en": "Biosafety Organizations"
```
Comment (blaisefonseca) [inline app/app-data/abs/realm-configuration.json:208]: wrong file

File: app/app-data/bch/realm-configuration.json [hunk 3]
```diff
@@ -304,7 +373,10 @@
           "description": {
             "en": "Information on capacity-building initiatives at national, regional and international levels, including activities and outcomes."
           },
-          "shortCode": "CBA"
+          "shortCode": "CBA",
+          "titlePlural": {
+            "en": ""
```
Comment (blaisefonseca) [inline app/app-data/bch/realm-configuration.json:378]:
```suggestion
            "en": "Capacity Building Initiatives"
```

File: app/app-data/bch/realm-configuration.json [hunk 4]
```diff
@@ -304,7 +373,10 @@
...
+          }
```
Comment (blaisefonseca) [inline app/app-data/bch/realm-configuration.json:379]: remove empty object
```suggestion
```

**Review by blaisefonseca — CHANGES_REQUESTED** [review]
please see comments

File: app/app-data/bch/realm-configuration.json (hunk 4 above)
Comment (tahmeedshah) [reply app/app-data/bch/realm-configuration.json]: corrected as suggested

File: app/app-data/bch/realm-configuration.json [hunk 5]
```diff
@@ -208,17 +250,23 @@
...
           "title": {
-            "en": "Biosafety Information Resource Centre"
+            "en": "Biosafety Virtual Library Resources"
           },
           "description": {
-            "en": "Biosafety Information Resource Centre"
+            "en": "An electronic catalogue of biosafety-related publications and information resources, including literature on biosafety, reports and case studies, awareness-raising materials, videos, capacity-building resources, etc."
           },
-          "shortCode": "BIRC"
+          "shortCode": "BVLR",
```
Comment (blaisefonseca) [inline app/app-data/bch/realm-configuration.json:266]:
```suggestion
          "shortCode": "VLR",
```

**Review by blaisefonseca — APPROVED** [review]

**Pull request merged**